styles/_top-nav.scss

```diff
@@ -0,0 +1,64 @@
+.top-nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  text-transform: uppercase;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  padding: 0 25%;
```

Comment:
Instead of using the `padding` in percent, look at using `max-width` along with `margin: auto`